.eslintrc.js

```diff
@@ -1,4 +1,100 @@
-// cf: https://decodenatura.com/set-up-react-native-typescript-eslint-prettier/
 module.exports = {
-  extends: 'universe/native',
+  extends: ['plugin:jsx-a11y/recommended', 'plugin:prettier/recommended'],
```

Comment:
This directly mimics the `butcherbox-frontend` eslint config, which is also a Gatsby app ✨